packages/editor/src/components/post-publish-panel/maybe-tags-panel.js

```diff
@@ -60,7 +60,7 @@ export default compose(
 	withSelect( ( select ) => {
 		const postType = select( 'core/editor' ).getCurrentPostType();
 		const tagsTaxonomy = select( 'core' ).getTaxonomy( 'post_tag' );
-		const tags = select( 'core/editor' ).getEditedPostAttribute( tagsTaxonomy.rest_base );
+		const tags = tagsTaxonomy && select( 'core/editor' ).getEditedPostAttribute( tagsTaxonomy.rest_base );
 		return {
 			areTagsFetched: tagsTaxonomy !== undefined,
 			isPostTypeSupported: tagsTaxonomy && tagsTaxonomy.types.some( ( type ) => type === postType ),
```

Comment:
FWIW on the line below this one we're passing `hasTags`, which reads as though it's a boolean value, but following the `&&` and its behavior of using the lefthand value, we're likely passing `null` or `undefined` when there are no tags, not `false`.

Making a point of it here only because this tends to be overlooked in applying the `&&` pattern for conditions, added here.